plugins/compliance-hub/frontend/public/templates/userConsentHistory.html

```diff
@@ -81,7 +81,7 @@
                     </template>
 
                 </el-table-column>
-                <el-table-column width="230" sortable="custom" prop="ts" label="TIME">
```

Comment:
Instead of removing you could make this min-width @shaharyar-shamshi

Reply:
yeah that would be better.

Reply:
done

Reply:
@itsiprikshit  @kanwarujjaval